packages/@react-stately/toast/test/useToastState.test.js

```diff
@@ -30,6 +30,8 @@ describe('useToastState', () => {
     let {result} = renderHook(() => useToastState());
     expect(result.current.toasts).toStrictEqual([]);
     act(() => result.current.onAdd(newValue[0].content, newValue[0].props));
+    // timeoutId is dynamic internal so removing it
+    delete result.current.toasts[0].timeoutId;
```

Comment:
I don't understand the code comment or why this is being done

Reply:
If this is to prevent setTimeouts from firing, you should probably be making use of jest useFakeTimers and should be running them out

Reply:
the timeout is firing and it adds a dynamic value to the array... I've removing that value

Reply:
why not let the timeout run in a controlled way? this seems like a test prone to breaking or at least not capable of catching real world use cases since it's doing something that no code we use would ever do. unless it's expected for these to be deleted?

Reply:
I rewrote these tests and next time I run into this issue I won't use `toStrictEqual()` and will use `objectContaining()` to ignore a value.